calcom/cal.com
# fix: app store cli windows compatibility issue #10324

## What does this PR do?

This PR addresses compatibility issues related to Windows environments in the CLI commands for **creating, deleting, and modifying** applications. It ensures that the CLI commands work correctly on both Windows and Unix-based systems. The changes made in this PR aim to resolve the issue reported in #10324.

Fixes #10324

No visual changes are expected on the application.

## Requirement/Documentation

N/A

## Type of change


## How should this be tested?

The changes in this PR should be tested as follows:

- The CLI commands for creating, deleting, and modifying applications should be tested on both Windows and Unix-based systems to ensure compatibility.
- Set up test environments with different configurations (e.g., different database setups, different app categories) to verify that the CLI commands work as expected in various scenarios.
- Perform the usual tests for creating, modifying, and deleting applications using the CLI and verify that they complete successfully on both Windows and Unix-based systems.

## Mandatory Tasks
